### PR TITLE
Add NoVNC troubleshooting steps to manual

### PR DESCRIPTION
**Pull request by stbenjam**
Related to 3307.  

The mess of a diff in 3.1.2_firewall_configuration.md was due to sorting the ports in order.

### CODE REVIEW

File: _includes/manuals/1.4/7.1_novnc.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Foreman uses the excellent javascript VNC library noVNC, which allows clientless VNC within a web browser. When a console is opened by the user's web browser, Foreman opens a connection to TCP Port 5910 (and up) on the hypervisor and redirects that to a TCP Port 5910 (and up) on itself.
+
+### Known issues
+
+* Currently only unencrypted connections are possible, even if you connect to Foreman via HTTPS.
+* Keyboard mappings are currently fixed to English only.
+* When using Firefox, if you use Foreman via HTTPS, Firefox might block the connection (see limiations below), to fix it, goto about:config and enable network.websocket.allowInsecureFromHTTPS
```
Comment (domcleal) [inline _includes/manuals/1.4/7.1_novnc.md]: Limitations section wasn't copied from the wiki, remove that reference, or include it?

**Comment by domcleal** [comment]
The explanation of the network connections from the requirements section might be useful too?

**Comment by stbenjam** [comment]
Ok will do

**Comment by stbenjam** [comment]
@domcleal How's this?

It's not the exact wiki page because I was trying to clean it up and make it clearer (e.g. "no firewall" was not really a good requirement).

**Comment by domcleal** [comment]
Looks good, thanks @stbenjam!

**Pull request merged**